Altis_Life.Altis/core/init.sqf

```diff
@@ -45,15 +45,14 @@ diag_log "::Life Client:: Waiting for the server to be ready..";
 waitUntil{!isNil "life_HC_isActive"};
 if (life_HC_isActive) then {
     waitUntil{!isNil "life_HC_server_isReady" && !isNil "life_HC_server_extDB_notLoaded"};
-    _server_isReady = life_HC_server_isReady;
+    waitUntil{life_HC_server_isReady};
```

Comment:
The HC isReady is only true IIRC, so we shouldn't need to await this one, but the other is needed :)